app/misc/repo.ts

```diff
@@ -258,11 +259,85 @@ function openRepository() {
     }
   }
 
+// works as a monitor for any change to the reference list
+function refreshList() {
```

Comment:
Is it possible to call refreshList() from refreshAll() so we don't have duplicated code for refreshing the list inside refreshAll()?

Reply:
@sterlinglaird yes, I think it is a good idea and will look into it.